OWNERS.yml

```
@@ -0,0 +1,4 @@
+filters:
+  "CHANGELOG.md":
+  approvers:
+    - tech-writers
```

```suggestion
    approvers:
      - tech-writers
```

(the `approvers` directive must be indented to apply it to the filter you've set up)